### PR TITLE
Add Chinese internationalization to zh-cn.json

### PR DESCRIPTION
**Pull request by feast107**

### CODE REVIEW

**Comment by takkuumi** [comment]
@feast107     fucking great

**Comment by feast107** [comment]
> @feast107他妈的很棒

Let's Go

File: client/src/assets/i18n/zh-cn.json [hunk 1]
```diff
@@ -604,64 +1134,115 @@
...
+    "msg.report-property-missing-value": "有不正确或缺失的值！",
+
+    "msg.home-welcome": "等待......去编辑器，映射设备，设计你的可视化并绑定设备的变量",
     "msg.project-load-error": "无法读取'{{value}}'",
-    "msg.project": "是否保存项目？",
-    "msg.tags-remove-all": "是否删除所有变量？",
-    "msg.view-remove": "是否删除视图'{{value}}'？",
+    "msg.project": "您想保存项目的变化吗？",
+    "msg.tags-remove-all": "您想删除所有标签吗？",
+    "msg.view-remove": "您是否想删除视图'{{value}'？",
```
Comment (takkuumi) [inline client/src/assets/i18n/zh-cn.json:1222]: should be `您是否想删除视图'{{value}}'？`

Comment (feast107) [reply client/src/assets/i18n/zh-cn.json]: Right, I'v fix this error now.

**Comment by unocelli** [comment]
Great Job, Thanks a lot

**Pull request merged**